src/util/deleteThought.ts

```diff
@@ -37,17 +35,18 @@ import {
   prevSibling,
   splitChain,
 } from '../selectors'
+import { Child, Thought } from '../types'
 
 /** Deletes a thought. */
 export const deleteThought = () => {
 
   const state = store.getState()
-  const path = state.cursor
+  const path:(Thought[] | Child[]) = state.cursor
```

Comment:
Why use `Child[]` when we have a `Path` type? Also, why `Thought[]` at all? A `Thought` is an entry in the `thoughtIndex` which is not related to a `Path`.